docs/serving/batch-serving.md

```diff
@@ -0,0 +1,611 @@
+        vertical-align: middle;
+    }
+
+    .dataframe tbody tr th {
+        vertical-align: top;
+    }
+
+    .dataframe thead th {
+        text-align: right;
+    }
+</style>
+<table border="1" class="dataframe">
+  <thead>
+    <tr style="text-align: right;">
+      <th></th>
+      <th>1</th>
+      <th>3</th>
+      <th>4</th>
+      <th>labels</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th>0</th>
+      <td>-1.932260</td>
+      <td>-1.863833</td>
+      <td>-1.045635</td>
+      <td>1</td>
+    </tr>
+    <tr>
+      <th>1</th>
+      <td>-2.835548</td>
+      <td>-1.385303</td>
+      <td>-2.276456</td>
+      <td>0</td>
+    </tr>
+    <tr>
+      <th>2</th>
+      <td>-1.807549</td>
+      <td>-1.118860</td>
+      <td>1.104900</td>
+      <td>0</td>
+    </tr>
+    <tr>
+      <th>3</th>
+      <td>-0.345659</td>
+      <td>-0.046066</td>
+      <td>0.447280</td>
+      <td>0</td>
+    </tr>
+    <tr>
+      <th>4</th>
+      <td>-1.746045</td>
+      <td>-1.110985</td>
+      <td>-1.675466</td>
+      <td>0</td>
+    </tr>
+  </tbody>
+</table>
+</div>
+
+
+### View the results in the UI 
+
+The output is saved as a parquet file under the project artifact path.
+in the UI you can go to the prediction job --> artifact tab to view the details
+
+<img src="../_static/images/prediction-results.png" alt="Jobs" width="1400"/>
+
+[Back to the top](#XGBoost-trainer)
+
+### How to scehdule a batch job
+
+To schedule a job, you can set the schedule parameter of the run method. The scheduling is done by using a crontab format.
+
+You can also schedule jobs from the dashboard. On the Projects > Jobs and Workflows page, you can create a new job using the New Job wizard. At the end of the wizard you can set the job scheduling. In the following example, the job is set to run every 30 minutes.
+
+
+```python
+predict_run = auto_trainer.run(
+    params={
+        "dataset": sample,
+        "model": model_path,
+        "drop_columns": [0, 2],
+        "label_columns": "labels",
+    },
+    handler="predict",
+    local=False,
+    schedule='*/30 * * * *'
+)
+```
+
+> 2022-08-11 06:01:09,183 [info] starting run auto-trainer-predict uid=9310360c64744c29991d1145d04d55e5 DB=http://mlrun-api:8080
+> 2022-08-11 06:01:09,456 [info] task scheduled, {'schedule': '*/30 * * * *', 'project': 'batch-inference-admin', 'name': 'auto-trainer-predict'}
+
+
+## Viewing the drift result
+
+By default, the auto_trainer functions analyze drift by comparing the input and the sample dataset statistics. <br> 
+it triggers a drift analysis job which generates 3 output files that can be viewed under the artifacts tab for the prediction job
+* drift_table_plot - shows expected vs actual statistics for every feature as a plot.
+* drift_results - shows the drift result per feature as a JSON format so you can use it to process this data and apply your logic on top of it.
+* dataset_statistics - shows the statistics of the input dataset.
```

Comment:
Should also show how to present the artifact here

Reply:
should be done with something like:

```
predict_run_object.artifact('drift_table_plot').show()
```

In the next section Drift table plot example